src/Transport/Creation/AzureServiceBusSubscriptionCreator.cs

```diff
@@ -101,6 +101,9 @@ public async Task<SubscriptionDescription> Create(string topicPath, string subsc
 
         public async Task DeleteSubscription(string topicPath, string subscriptionName, SubscriptionMetadataInternal metadata, string sqlFilter, INamespaceManagerInternal namespaceManager, string forwardTo)
         {
+            _ = forwardTo;
```

Comment:
Since this isn't a public class, could we just remove these parameters instead?

Reply:
We could and for other repositories, it would make sense. This one is deprecated in a few months. Is it worth it? 🙂

Reply:
Good point. Probably not worth it!